internal/quaiapi/quai_api.go

```diff
@@ -880,21 +881,12 @@ func (s *PublicBlockChainQuaiAPI) GetPendingEtxsFromSub(ctx context.Context, raw
 		return nil, err
 	}
 	fields := make(map[string]interface{})
-	fields["header"] = pEtxs.Header.RPCMarshalHeader()
+	fields["header"] = pEtxs.Header.RPCMarshalWorkObject()
 	fields["etxs"] = pEtxs.Etxs
 
 	return fields, nil
 }
 
-func (s *PublicBlockChainQuaiAPI) SetSyncTarget(ctx context.Context, raw json.RawMessage) error {
```

Comment:
Why remove this?

Reply:
we dont need it but will remove it in a separate commit and also remove all the reference, this is just 10% removal of the setSyncTarget